app/Filament/Pages/EditProfile.php

```diff
@@ -92,6 +92,9 @@ public function form(Form $form): Form
     {
         /** @var User $user */
         $user = auth()->user();
+        $retentionLicense = $user->hasLicense(LicenseType::RetentionCrm);
+        $recruitmentLicense = $user->hasLicense(LicenseType::RecruitmentCrm);
+        $crmSetting = ($retentionLicense || $recruitmentLicense) ? true : false;
```

Comment:
Please use `->hasAnyLicense([` instead and maybe call the variable something like `$hasCrmLicense` so it's clear what it's referencing.

Reply:
@mxm1070 Made the changes.

Reply:
@jayushi-canyon `hasAnyLicense` takes an array so you can do: 
```php
$hasCrmLicense = $user->hasAnyLicense([LicenseType::RetentionCrm, LicenseType::RecruitmentCrm]);
```

Also you can remove the calls to 
```php
->lockedWithoutAnyLicenses(user: auth()->user(), licenses: [LicenseType::RetentionCrm, LicenseType::RecruitmentCrm]),
```
because it looks like they're all covered by `->visible($hasCrmLicense)` now.

Reply:
@mxm1070 Done with the changes